synfig-studio/src/gui/canvasview.cpp

```diff
@@ -596,6 +597,10 @@ CanvasView::CanvasView(etl::loose_handle<Instance> instance,etl::handle<CanvasIn
 	keyframe_dialog          (*App::main_window,canvas_interface_),
 	preview_dialog           ()
 {
+	// Make this toolbar small for space efficiency
+	auto style_context = this->get_style_context();
+	style_context->add_class("synfigstudio-efficient-workspace");
```

Comment:
Do we need to add a class to the canvas view as a whole or just a specific toolbar?

Reply:
It's safe to set it as a whole. Otherwise, I'd have to duplicate these lines down at least two times inside CanvasView:
1. For top toolbar
2. For bottom toolbar
3. And additionally the tab bar (Wasn't styled earlier...)

Remember DRY? I think setting class as a whole is much better. We're styling only buttons and it's image.
```css
class .synfigstudio-efficient-workspace {
    button        : <value>;
    button > image: <value>;
}
```